ChangeLog.md

```diff
@@ -7,6 +7,8 @@
     (Pablo Hernandez-Cerdan [#1529](https://github.com/DGtal-team/DGtal/pull/1529))
   - Modernize CMake: Avoid global includes and links, use `target_` commands instead
     (Pablo Hernandez-Cerdan, David Coeurjolly [#1524](https://github.com/DGtal-team/DGtal/pull/1524))
+  - Modernize CMake: Prefer use targets than directories and libraries
```

Comment:
```suggestion
  - Modernize CMake: Prefer use targets rather than directories and libraries
```

Reply:
Thanks, fixed. I also removed linking to the list of libraries `${DGtalLibDependencies}` in examples. Those libraries are in the DGtal target already.